### ISSUE TITLE
Sogo CalDAV / CardDAV sync not working

### ISSUE THREAD

**Issue by vsatmydynipnet** [YunoHost-Apps/sogo_ynh#16]
Hi,

first of all thank you for this really cool peace of software. I have setup [YunoHost](https://yunohost.org/) 4.3.6.3 (stable) for testing today. Tested some Apps, like Jitsi, Matrix Synapse, Wordpress, all fine. 

My Problem is with Sogo:

- Sogo by itself works fine, but impossible to sync Calendar and Contacts via Dav, e.g. from Thunderbird. The log shows all 302 for the requests, where my Sogo on iRedmail schows 207 and 200 responses. Iredmail is already on Sogo5, but it worked with Sogo 4 too. Comparing the configs brought me to no result.

I have setup this for a friend who wants to go with self-hosting, but without Thunderbird  and DavX Sync possibility it is impossible to use it.  

Is there any known workaround t make syncing possible? Would love to see him running his own Sogo , Wordpress, Synapse and Jitsi Server.

Thank you for your time and any hint.

Greetings

**Comment by vsatmydynipnet** [comment]
After debugging nginx i found out the visitors must have not only the Sogo Sync Client permission, they must also have Sogo permission if it is normally handled vie SSO.

All good now.

**Comment by Josue-T** [comment]
Do you have the URL which was blocked by the SSO ?

**Comment by Josue-T** [comment]
Closing as no more information

**Comment by vsatmydynipnet** [comment]
It looks like  this is still a problem. I needed to set SOGO permissions to guest on a fresh install to make caldav/carddav working. Otherwise Thunderbird has not found the calendar and/or addressbook.

**Comment by Josue-T** [comment]
seem a duplicate of #34
